packages/gatsby-plugin-page-creator/src/path-utils.ts

```diff
@@ -121,3 +123,9 @@ export function compose(
   return (filePart: string): string =>
     functions.reduce((value, fn) => fn(value), filePart)
 }
+
+// Use globby to find all page files with collection routes
+export const pagesGlob = `**/**\\{*\\}**(/**)`
```

Comment:
Could be `**/**\\{*\\}**(/**)` or `['**/**\\{*\\}**/**','**/**\\{*\\}**']`, not sure which is preferable.

Reply:
With the comment above I think it's fine 👍